### ISSUE TITLE
Access the printer through the internet.

### ISSUE THREAD

**Issue by limbonic** [OctoPrint/OctoPrint#78]
Hello, 
When i install Octoprint with the default settings i can access my printer through localhost: 127.0.0.1:5000. Because i want to access it through the internet i need to do port forward to my router to the particular port (5000 for example) and that's it, or i need to add in the settings my ip address because after the port forwarding procedure i still cant access my  printer from the internet. I use no-ip application and a raspberry pi.

Thanks.

**Comment by stefi01** [comment]
tried to set it up, gave up on internet and wifi, might try that new software using a pi, see how that works as its suposed to be more straight forward.

**Comment by BobDavis73** [comment]
I use Google Remote Desktop to operate my OctoPi from anywhere.  Works great.  A bit of lag but that is to be expected.

**Comment by wescrockett** [comment]
The OctoPi distribution made it fairly simple to set up over port 443 using HAProxy as a reverse proxy. It runs really well with a Pi 3.

**Comment by Nathrelor** [comment]
I want to set up octoprint so I can control my printer over the internet without being on the same network so I can start, view, and, if necessary, stop or pause a print while I'm out. I set up octoprint anywhere and got that going easily, but that only lets me view, pause, or cancel prints. Would someone be able to help me set this up? I saw some mentions of port forwarding that I think might be useful but I'm by no means an expert on how to do that and as @foosel mentioned I would also like to use a reverse proxy (whatever that is lol) or something like it to protect my printer from being accessed by just anyone. I think @wescrockett may have set something up like what I'm looking for, but I'm not sure since idk exactly what his/her setup allows them to do.
